jbpm-db-scripts/src/main/resources/db/ddl-scripts/mysql5/mysql5-jbpm-schema.sql

```diff
@@ -689,17 +689,13 @@
         foreign key (task_id) 
         references Task (id);
 
+            
     alter table PeopleAssignments_PotOwners 
-        add index IDX_PAsPot_Entity (entity_id), 
+        add index IDX_PAsPot_TaskEntity (task_id, entity_id), 
         add constraint FK_tee3ftir7xs6eo3fdvi3xw026 
         foreign key (entity_id) 
         references OrganizationalEntity (id);
 
-    alter table PeopleAssignments_PotOwners 
-        add index IDX_PAsPot_Task (task_id), 
-        add constraint FK_4dv2oji7pr35ru0w45trix02x 
-        foreign key (task_id) 
-        references Task (id);
```

Comment:
Why removing foreign key and constraint?

Reply:
corrected